ops/framework.py

```diff
@@ -199,6 +199,11 @@ def __init__(self, parent, key):
         else:
             self.framework = parent.framework
             self.handle = Handle(parent, kind, key)
+        if parent is not self:
```

Comment:
Since `Framework` is the only case where `parent is self`, this is equivalent to `if not issubclass(parent, Framework):` a.k.a. the `else` clause immediately above, so it would be cleaner to remove this `if` and fold the body into the above `else`.

Reply:
I like Gustavo's suggestion above better.

Reply:
Actually, that isn't quite true. In one case it is "is parent a Framework" and the other is "is self a Framework". Because there are other objects (Framework._stored) whose immediate parent is Framework, but is not, itself, a Framework.
However, _track with 'if obj is self" is fine. But I did want to clarify that they aren't quite the same. (I also thought at first it might be, before realizing.)